test/templates/product-menu/component.njk

```diff
@@ -33,13 +33,14 @@ product_menu:
   }}
 </div>
 {%- else %}
-{{ product_menu_logo(logo=brand.logo, name=brand.name) }} <span class="fd-shellbar__title">{{ product.title }}</span>
+{# {{ product_menu_logo(logo=brand.logo, name=brand.name) }}  #}<span class="fd-shellbar__title">{{ product.title }}</span>
```

Comment:
commented?

Reply:
Done. I revisited the duplicate logo and opted to remove it.